### PR TITLE
Secure import errors with eslint-plugin-import

### PR DESCRIPTION
**Pull request by Metehan-Altuntekin**
Separated to a branch because of complexities faced along the way

### CODE REVIEW

**Comment by Metehan-Altuntekin** [comment]
@mike-hogan can you check if [this fix](https://github.com/cozemble/monorepo/pull/2/commits/9f82974f898d04f985e730666ef21d488e35f386) is correct?

File: model/api/src/dataRecordPathFns.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import {
-  DataRecordPath,
-  DataRecordPathElement,
-  Property,
+  type DataRecordPath,
+  type DataRecordPathElement,
+  type Property,
   propertyDescriptors,
 } from '@cozemble/model-core'
```
Comment (mike-hogan) [inline model/api/src/dataRecordPathFns.ts:7]: have you tested this mate?  I have a memory of svelte:check not liking this.  Sorry not this case, the case above.  If all the imported things are `type` I think `svelte:check` wants `import type {....}`

**Review by mike-hogan — COMMENTED** [review]
So happy you are doing this mate.  If it works (see my other comment) then I am all for it!

**Comment by mike-hogan** [comment]
> @mike-hogan can you check if [this fix](https://github.com/cozemble/monorepo/pull/2/commits/9f82974f898d04f985e730666ef21d488e35f386) is correct?

Yes it is correct

File: model/api/src/dataRecordPathFns.ts (hunk 1 above)
Comment (Metehan-Altuntekin) [reply model/api/src/dataRecordPathFns.ts:7]: Yes, no problems from build, IDE or linting. `propertyDescriptors` is not a type

**Pull request merged**